src/objects/zcl_abapgit_object_w3xx_super.clas.abap

```diff
@@ -289,44 +289,7 @@ CLASS zcl_abapgit_object_w3xx_super IMPLEMENTATION.
       zcx_abapgit_exception=>raise( 'Cannot upload W3xx data' ).
     ENDIF.
 
-    CONCATENATE 'W3' ms_key-relid INTO lv_tadir_obj.
```

Comment:
it needs this concatenate?

Reply:
constructor has `ms_key-relid = ms_item-obj_type+2(2).` turning `W3MI` into `MI` which the concat just reverses. not necessary.